engine/src/main/java/org/terasology/identity/storageServiceClient/HttpMethod.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.identity.storageServiceClient;
+
+/**
+ * Http methods; only the relevant ones for now but can easily be extended with the other standard ones.
+ */
+enum HttpMethod {
```

Comment:
Ok, adding all the methods according to https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods